docs/content/preview/yugabyte-platform/anywhere-automation/anywhere-cli.md

```diff
@@ -0,0 +1,269 @@
+  By default, this writes the value to the CLI configuration file `$HOME/.yba-cli.yaml`.
+
+  You can create multiple configuration files, and switch between them using the [--config](#global-flags) flag. You can add any of the other [global flags](#global-flags) to your configuration files.
+
+- Using [environment variables](#environment-variables). Environment variables must begin with `YBA_`. For example:
+
+  ```sh
+  export YBA_APITOKEN=AWERDFSSS
+  yba universe list
+  ```
+
+If a value is set in an environment variable and a configuration file, the environment variable takes precedence. Setting the value using the flag takes precedence over both.
+
+### Environment variables
+
+You can set the following CLI environment variables.
+
+| Variable | Description |
+| :--- | :--- |
+| YBA_CI | Set to `true` to avoid outputting unnecessary log lines. |
+| YBA_AWS_SECRET_KEY | AWS secret access key. For encryption at rest of AWS clusters. |
+
+## Syntax
+
+```sh
+yba [-h] [ <resource> ] [ <command> ] [ <flags> ]
+```
+
+- resource: resource to be changed
+- command: command to run
+- flags: one or more flags, separated by spaces.
+
+For example:
+
+```sh
+yba universe list
+```
+
+### Online help
+
+Access command-line help for the CLI by running the following command:
+
+```sh
+yba help
+```
+
+For help with specific resource commands, use the `--help` or `-h` flag, in the following form:
+
+```sh
+yba [ resource ] [ command ] -h
+```
+
+For example, to print the command-line help for the cluster `create` command, run the following:
+
+```sh
+yba universe list -h
+```
+
+Print the version of the CLI:
+
+```sh
+yba --version
+```
+
+## Commands
+
+You can manage the following resources:
+
+| Resource | Description |
+| :--- | :--- |
+| [login](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_login.md) | Connect to YugabyteDB Anywhere host machine using email and password. |
+| [provider](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_provider.md) | Manage provider configurations. |
+| [register](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_register.md) | Register a YugabyteDB Anywhere customer. |
+| [storage-config](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_storage-config.md) | Manage storage configurations. |
+| [task](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_task.md) | Manage tasks. |
+| [universe](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_universe.md) | Manage universes. |
+| [yb-db-version](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_yb-db-version.md) | Manage YugabyteDB releases. |
+
+Use the following commands to configure the CLI:
+
+| Resource | Description |
+| :--- | :--- |
+| [auth](https://github.com/yugabyte/yugabyte-db/blob/master/managed/yba-cli/docs/yba_auth.md) | Write API token to a configuration file. |
```

Comment:
It would be better to put login as a configuration command as well. Since either `login` or `auth` can be used to connect to the YBA host.